app/src/main/java/ch/epfl/skysync/components/BackButton.kt

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.skysync.components
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import ch.epfl.skysync.ui.theme.lightOrange
+
+/**
+ * Composable function representing a back button with an arrow icon and text.
+ *
+ * @param BackClick Callback function to be invoked when the back button is clicked.
+ */
+@Composable
+fun backbutton(BackClick: () -> Unit) {
```

Comment:
Composable name are capitalized, you should call it `BackButton`.
However, callback arguments are not, here `BackClick` could be call `onClick`.